Strings/ZFunction.java

```diff
@@ -0,0 +1,29 @@
+package Strings;
```

Comment:
Kindly add a description of your code as this repository is for learning purpose

Reply:
Okey

Reply:
Is there any standards that this repo follows? If so can you show me an example of the description so I meet that standards.

Reply:
You can refer to this one. https://github.com/TheAlgorithms/Java/pull/1352/files

Reply:
Thanks @deadshotsb I have added description and link of the algorithm review

Reply:
Sorry for the inconvenience, @vakhokoto, kindly give a desciption instead of providing links, you can give an example case if needed.